tests/unit/hubble/test_helper.py

```diff
@@ -44,19 +44,31 @@ def test_unpack_package(tmpdir, dummy_zip_file):
     helper.unpack_package(dummy_zip_file, tmpdir / 'dummp_executor')
 
 
-first_time = True
+first_time = False
 
 
-def test_disk_cache():
-    @disk_cache((Exception,))
+def test_disk_cache(tmpdir):
+    global first_time
```

Comment:
what is `first_time` needed? Isn't there a better way? `globals` are scary

Reply:
but u do not need a `global`. U can have a local variable inside the `test` and manipulate it there

Reply:
Can't access it : UnboundLocalError: local variable 'first_time' referenced before assignment

Reply:
never mind I know now why that shows up

Reply:
done